zk-rs/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "@lightprotocol/zk-rs",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "build": "wasm-pack build --target nodejs",
+    "test": "ts-mocha --resolveJsonModule ./tsconfig.json -t 100000000 tests/wasm.test.ts --exit",
+    "test-wasm": "cargo test -- --nocapture"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
```

Comment:
couldn't all of these dependencies be dev dependencies?

Reply:
It's not important, because we don't import this lib anywhere, it's ephemeral package that builds another package with wasm inside. 
Anyway, I agree, it would be more accurate. 
Fixed.